bigbluebutton-html5/imports/ui/components/chat/message-form/styles.scss

```diff
@@ -62,6 +62,10 @@
   border: 1px solid $color-gray-lighter;
   box-shadow: 0 0 0 2px $color-gray-lighter;
 
+  @include mq($small-only) {
+    font-size: 16px;
```

Comment:
You need to use `rem` for font-sizes otherwise the font-size option will not work on this component.

Reply:
When I googled how to disable automatically zoom in the input box on iPhone screen, others (of StackOverflow)  recommended that the font size should be bigger than 16px. Is it ok to use rem than px? I guess if I use rem for the font-size, the font-size can be smaller than 16px according to the root element's font-size.